src/rpc/net.cpp

```diff
@@ -94,6 +94,7 @@ static RPCHelpMan getpeerinfo()
                             {RPCResult::Type::STR, "addr", "(host:port) The IP address and port of the peer"},
                             {RPCResult::Type::STR, "addrbind", "(ip:port) Bind address of the connection to the peer"},
                             {RPCResult::Type::STR, "addrlocal", "(ip:port) Local address as reported by the peer"},
+                            {RPCResult::Type::STR, "network", "Network (ipv4, ipv6, or onion) the peer connected through"},
```

Comment:
bebd062eef3fa2d33c9216c368977dbedb70a443

While here, mind making "addrbind" and "addrlocal" ordering consistent for help and output?

Reply:
Good idea. Done.